src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static String DEFAULT_LOGIN = "login";
+    private static String DEFAULT_PASSWORD = "password";
+    private static Integer DEFAULT_AGE = 18;
```

Comment:
Consider using `final` for `DEFAULT_LOGIN`, `DEFAULT_PASSWORD`, and `DEFAULT_AGE` to indicate that these are constants and should not be modified.